tests/config.js

```diff
@@ -95,64 +119,154 @@ describe( 'set', () => {
 			}
 		} );
 
-		expect( config ).to.have.property( 'resize' );
-		expect( config.resize ).to.have.property( 'minheight' ).to.equal( 400 );
-		expect( config.resize ).to.have.property( 'maxheight' ).to.equal( 800 );	// Not touched
-		expect( config.resize ).to.have.property( 'hidden' ).to.equal( true );
+		expect( config.get( 'resize' ) ).to.be.eql( {
```

Comment:
I treat `eql` as short version of `deep.equal`
http://chaijs.com/api/bdd/#method_eql